src/auth/auth.controller.ts

```diff
@@ -0,0 +1,13 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { UserCredentialsDto } from '../user/dto/user.credentials.dto';
+import { AuthService } from './auth.service';
+
+@Controller('')
```

Comment:
Unacceptable to leave it empty, use the name of the controller here. so you can have /api/auth/register or /api/auth/login